apps/server/src/iam/decorators/active-user.decorator.ts

```
@@ -0,0 +1,9 @@
+import { createParamDecorator, ExecutionContext } from '@nestjs/common';
+import { ActiveUserData } from '../interfaces/active-user-data.interface';
+export const ActiveUser = createParamDecorator(
+  (field: keyof ActiveUserData | undefined, ctx: ExecutionContext) => {
+    const request = ctx.switchToHttp().getRequest();
+    const user: ActiveUserData | undefined = request.user;
+    return field ? user && user[field] : user;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)